tests/user_settings/src/main.c

```diff
@@ -130,6 +133,35 @@ ZTEST(user_settings_suite, test_settings_default_value)
 	zassert_equal(value_out, value, "Value should be unchanged");
 }
 
+ZTEST(user_settings_suite, test_settings_default_value_twice)
+{
+	int err;
+
+	/* Set default value for t5 */
+	uint32_t default_value = 69;
+
+	err = user_settings_set_default_with_id(5, &default_value, sizeof(default_value));
+	zassert_ok(err, "set default should not error here");
+
+	/* Set default with same value should not error */
+	err = user_settings_set_default_with_id(5, &default_value, sizeof(default_value));
+	zassert_ok(err, "set default should not error here");
+
+	/* Set default with new value should error if CONFIG_USER_SETTINGS_DEFAULT_OVERWRITE=n.
+	 *
+	 * NOTE: If running these test with Twister, both cases will be tested.
+	 * The command is: east twister -T . -p native_posix
```

Comment:
This comment is not needed here. The tests should be run with the `make test ` command, which doesn't need to be explicitly mentioned here.

Reply:
That would be true if this project was up to date with the template. Which it is not :(

Reply:
#56